.golangci.yml

```diff
@@ -40,10 +39,13 @@ issues:
     - text: "ST1016:"
       linters:
         - stylecheck
-    - path: "migrations"
-      text: "SA1019:"
+    - text: "SA1019:"
       linters:
         - staticcheck
+    - path: "crypto/keyring"
+      text: "G306:"
+      linters:
+        - gosec
```

Comment:
Some file has 0500 privileges and the lint was expecting 0600 or higher, seems unhelpful.